src/cxxdemangler/cxxdemangler_stdlib.cpp

```diff
@@ -4,9 +4,15 @@
 
 namespace bpftrace {
 
-char* cxxdemangle(const char* mangled)
+std::string cxxdemangle(const char *mangled)
```

Comment:
Nit: feels like the updates to the cxxdemangle paths should be a separate commit.